src/features/DrugChart/DrugChart.spec.jsx

```diff
@@ -16,127 +16,97 @@ jest.mock("../Calendar/Calendar", () => {
   };
 });
 
+const mockRegisterPane = jest.fn();
+const mockUnregisterPane = jest.fn();
+
+jest.mock("react-scroll-sync-hook", () => {
+  return function useScrollSync() {
+    return {
+      registerPane: mockRegisterPane,
+      unregisterPane: mockUnregisterPane,
+    };
+  };
+});
+
 const drugChartData = [
```

Comment:
can we move these mockData into a separate file. Something similar is done for DrugChartModal test cases - https://github.com/Bahmni/openmrs-module-ipd-frontend/blob/main/src/entries/DrugChartModal/DrugChartModalTestUtils.js